game/graphics/pipelines/opengl.cpp

```diff
@@ -529,6 +591,12 @@ void GLDisplay::render() {
     ImGui_ImplOpenGL3_RenderDrawData(ImGui::GetDrawData());
   }
 
+  // switch vsync modes, if requested
+  if (Gfx::g_global_settings.vsync != Gfx::g_global_settings.old_vsync) {
```

Comment:
why move this (just curious)?

Reply:
This was an accident, I was moving chunks code around so I could better visualize what was going on in the source code and this change must have snuck into my commit.